src/frontend/src/components/folderSidebarComponent/components/sideBarFolderButtons/index.tsx

```diff
@@ -67,6 +67,10 @@ const SideBarFoldersButtonsComponent = ({
 
   const handleUploadFlowsToFolder = () => {
     createFileUpload().then((files: File[]) => {
+      if (files?.length === 0) {
+        return;
```

Comment:
Maybe set notice

Reply:
I think this is not the case.. the user just pressed "Cancel" and the popup was been activated

Reply:
I see